crates/turbopack-css/src/asset.rs

```diff
@@ -216,17 +227,6 @@ impl CssChunkItem for ModuleChunkItem {
                 }
             });
 
-            // remove imports
```

Comment:
The CSS returned per module must not have `@import`s. Because the CSS is concatenated on chunk level and the `@import`s won't be at the top of the file anymore. You need to return the `@import` separately from the CSS code to allow the chunk code generation to hoist them to the top. There is a `CssChunkItemContent.imports` prepared for that.

With this in mind you also don't need to try to place the `@import` after the last `@import` during code generation. You can add an `imports` property to `CodeGeneration` and aggregate them in the `for code_gen in code_gens` loop above.